Improved: adding release info (OFBIZ-11450)

PierreSmits opened the pull request:
Also (re)setting maintainer to the PMC of the project

PierreSmits committed:
Improved: adding release info (OFBIZ-11450

Also (re)setting maintainer to the PMC of the project

JacquesLeRoux commented:
Hi Pierre,

Your changes look good to me, but I wonder if, for responsability reason, we don't need real person/s for <maintainer><foaf:Person> tags. Do you have experience in this?

PierreSmits commented:
Please put your comments in the ticket as it is still the collaboration tool of choice of the project.

PierreSmits committed:
Improved: maintainers email address

(OFBIZ-11450)

Thanks to Michael Brohl, Jacopo Cappellato and Jacques
Le Roux for feedback

JacquesLeRoux commented:
I wanted to squash and merge but it seems it does not rebase so I'll rather take the 2 commits and rebase
